mcc/odin/odin/odin.go

```diff
@@ -38,17 +39,15 @@ func (params CreateDBParams) GetRestoreDBInstanceFromDBSnapshotInput(
 	svc rdsiface.RDSAPI,
 ) (
 	restoreDBInstanceFromDBSnapshotInput *rds.RestoreDBInstanceFromDBSnapshotInput,
+	err error,
 ) {
 	var snapshot *rds.DBSnapshot
-	var err error
 	if params.OriginalInstanceName == "" {
-		log.Fatalf(
-			"Original instance is required",
-		)
+		return nil, fmt.Errorf("Original instance is required")
 	}
 	snapshot, err = GetLastSnapshot(params.OriginalInstanceName, svc)
 	if err != nil {
-		log.Fatalf(
+		return nil, fmt.Errorf(
```

Comment:
not sure about go, but in most languages multiple return values are considered bad practice...